src/adapters/humctl/HumctlAdapter.ts

```diff
@@ -88,11 +88,20 @@ export class HumctlAdapter implements IHumctlAdapter {
       result.stderr.includes('HTTP-403')
     ) {
       throw new UnauthorizedError();
-    } else if (result.stderr.includes('environment is required')) {
+    } else if (
+      result.stderr.includes('environment is required') ||
+      result.stderr.includes('missing env')
```

Comment:
Where are these new errors coming from?

Reply:
CLI returns unconsistent error messages and I found out that in most places it returns: `environment is required`, but in some: `missing env`